.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+/kind bug
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- Cluster-api-provider-vsphere version: 
+- Kubernetes version: (use `kubectl version`): 
+- OS (e.g. from `/etc/os-release`): 
```

Comment:
Can we also ask for adding CAPV and CAPI controller logs? that way it will be easier to debug any issue?

Reply:
Hi @sidharthsurana,

Since we don't have great instructions on how to obtain those yet, how about we get this PR merged so we're in sync with community standards and then we can enhance our asks later?

Reply:
feel free to cancel hold